source/features/infinite-scroll.tsx

```diff
@@ -29,10 +29,8 @@ const inView = new IntersectionObserver(([{isIntersecting}]) => {
 });
 
 function init(): Deinit {
-	const selectorObserver = observe('.ajax-pagination-btn', {
-		add(button) {
-			inView.observe(button);
-		},
+	observe('.ajax-pagination-btn', button => {
```

Comment:
I think this should use the `signal` passed to `init`

Reply:
That's new, I will use it